.vscode/settings.json

```diff
@@ -1,8 +1,6 @@
 {
   "editor.formatOnSave": true,
-  "git.mergeEditor": true,
```

Comment:
I don't enjoy using the 3-way merge editor so I disabled this. Also `eslint.autoFixOnSave` was apparently deprecated